nebula_common/include/nebula_common/continental/continental_srr520.hpp

```diff
@@ -0,0 +1,419 @@
+    os << ", ";
+    os << "lateral: " << arg.lateral;
+    os << ", ";
+    os << "vertical: " << arg.vertical;
+    os << ", ";
+    os << "yaw: " << arg.yaw;
+    os << ", ";
+    os << "pitch: " << arg.pitch;
+    os << ", ";
+    os << "plug_orientation: " << arg.plug_orientation;
+    os << ", ";
+    os << "length: " << arg.length;
+    os << ", ";
+    os << "width: " << arg.width;
+    os << ", ";
+    os << "height: " << arg.height;
+    os << ", ";
+    os << "wheel_base: " << arg.wheel_base;
+    os << ", ";
+    os << "max_distance: " << arg.max_distance;
+    os << ", ";
+    os << "frequency_slot: " << arg.frequency_slot;
+    os << ", ";
+    os << "cycle_time: " << arg.cycle_time;
+    os << ", ";
+    os << "time_slot: " << arg.time_slot;
+    os << ", ";
+    os << "hcc: " << arg.hcc;
+    os << ", ";
+    os << "power_save_standstill: " << arg.power_save_standstill;
+    os << ", ";
+    os << "sensor_ip_address0: " << arg.sensor_ip_address0;
+    os << ", ";
+    os << "sensor_ip_address1: " << arg.sensor_ip_address1;
+    os << ", ";
+    os << "configuration_counter: " << arg.configuration_counter;
+    os << ", ";
+    os << "status_longitudinal_velocity: " << arg.longitudinal_velocity_status;
+    os << ", ";
+    os << "status_longitudinal_acceleration: " << arg.longitudinal_acceleration_status;
+    os << ", ";
+    os << "status_lateral_acceleration: " << arg.lateral_acceleration_status;
+    os << ", ";
+    os << "status_yaw_rate: " << arg.yaw_rate_status;
+    os << ", ";
+    os << "status_steering_angle: " << arg.steering_angle_status;
+    os << ", ";
+    os << "status_driving_direction: " << arg.driving_direction_status;
+    os << ", ";
+    os << "characteristic_speed: " << arg.characteristic_speed_status;
+    os << ", ";
+    os << "radar_status: " << arg.radar_status;
+    os << ", ";
+    os << "temperature_status: " << arg.temperature_status;
+    os << ", ";
+    os << "voltage_status: " << arg.voltage_status;
+    os << ", ";
+    os << "blockage_status: " << arg.blockage_status;
+
+    return os;
+  }
+};
+
+using boost::endian::big_float32_buf_t;
+using boost::endian::big_uint16_buf_t;
+using boost::endian::big_uint32_buf_t;
+using boost::endian::big_uint64_buf_t;
+
+// CAN MSG IDS
+constexpr int RDI_NEAR_HEADER_CAN_MESSAGE_ID = 900;
+constexpr int RDI_NEAR_ELEMENT_CAN_MESSAGE_ID = 901;
+constexpr int RDI_HRR_HEADER_CAN_MESSAGE_ID = 1100;
+constexpr int RDI_HRR_ELEMENT_CAN_MESSAGE_ID = 1101;
+constexpr int OBJECT_HEADER_CAN_MESSAGE_ID = 1200;
+constexpr int OBJECT_CAN_MESSAGE_ID = 1201;
+constexpr int CRC_LIST_CAN_MESSAGE_ID = 800;
+constexpr int STATUS_CAN_MESSAGE_ID = 700;
+constexpr int SYNC_FUP_CAN_MESSAGE_ID = 53;
+constexpr int VEH_DYN_CAN_MESSAGE_ID = 600;
+constexpr int SENSOR_CONFIG_CAN_MESSAGE_ID = 601;
+
+// CRC IDS
+constexpr int NEAR_CRC_ID = 0;
+constexpr int HRR_CRC_ID = 1;
+constexpr int OBJECT_CRC_ID = 2;
+constexpr int TIME_DOMAIN_ID = 0;
```

Comment:
Where does `TIME_DOMAIN_ID` come from? I don't find it in the CAN doc

Reply:
Do you mean why is the `TIME_DOMAIN_ID` equal to 0?
Since there was no information, I just assumed it would be zero and it worked :innocent:

Reply:
@mojomex 
If you are satisfied with my reply, please mark is as `resolved` please !

Reply:
```suggestion
constexpr int TIME_DOMAIN_ID = 0; // Undocumented parameter, setting it to 0 seems to work fine.
```

Reply:
Added reference to the documentation in [a151918](https://github.com/tier4/nebula/pull/124/commits/a151918489d35f9901b291561e401d925803a9ab)